src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1086,10 +1090,39 @@ private void buildTableColumnHeaderForRecipientQuestionGiverView(List<ElementTag
                                                                        bundle.getResponseAnswerHtml(response, question),
                                                                        moderationButton);
             configureResponseRow(prevGiver, response.recipient, responseRow);
-            if (question.getQuestionDetails().isCommentsOnResponsesAllowed()) {
-                addCommentsToResponseRow(response, question, responseRow, userIndexesForComments);
-            } else {
-                responseRow.setCommentsOnResponsesAllowed(false);
+            String giverName = bundle.getNameForEmail(response.giver);
```

Comment:
Why not `addCommentsToResponseRow`? 

I think it won't affect other things but hard coded comment idx. Can you estimate the size of the changes?

Reply:
Or you can consider another way: Leave this side untouched and extract the code to common function. For GQR, pre-generated `userIndex` from `emailNameTable` and pass the map to the common function.

Reply:
> I think it won't affect other things but hard coded comment idx. Can you estimate the size of the changes?

It will probably affect 5k lines plus it would also mean making changes to existing UI tests which involve comment indexes.